src/styles/blocks/navigation.scss

```diff
@@ -0,0 +1,59 @@
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  text-align: center;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.nav {
+  margin-right: 50px;
+
+  &__list {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    margin: 0;
+    padding: 0;
+    list-style: none;
+    text-transform: uppercase;
+  }
+
+  &__item:not(:first-child) {
+    margin-left: 20px;
+  }
+
+  &__link {
+    display: block;
+    text-decoration: none;
+    line-height: 60px;
+    color: $color-text;
+
+    &:hover {
+      color: $color-accent-blue;
+    }
+  }
+}
+
+.logo__img {
+  display: flex;
+  justify-content: flex-start;
+  margin: 10px 0 0 50px;
+}
+
+.is-active {
```

Comment:
The `.is-active` class should be nested inside the `.nav__link` class for better organization. You can use `&.is-active` to achieve this.